src/06_multiple_futures/04_spawning.md

```diff
@@ -0,0 +1,16 @@
+# `Spawning`
+
+Spawning allows you to run a new asynchronous task in the background. This allows us to continue executing other code 
+while it is running.
+
+Say we have a web server that wants to accept connections without blocking the main thread. 
+To achieve this, we can use the `async_std::task::spawn` function to create and run a new task that handles the 
+connections. This function takes a future and returns a `JoinHandle`, which can be used to wait for the result of the 
+task once it's completed.
```

Comment:
It might be worth discussing some of the task life cycle questions around spawn. For example, what happens if main finishes before a the spawned task does in the example? Does the whole program block until the task is finished, or do the spawned tasks get cancelled?

Reply:
That section is now added with an example. Thanks for the feedback!